bcgov/moh-prime
# PRIME-2147 logout issue

Currently if you login for a user enrolment then logout you will get an error like the one below:

![image](https://user-images.githubusercontent.com/34261366/155244454-cf2051f3-16a2-4bdb-9095-ad31567888dd.png)

After the fix you should be redirected to the landing page for PRIME.

How to test:

Login for a user enrolment then log out. You shouldn't see the error screen in the screenshot above.

For the logout part, we see the error in the client logs. the errors were related to trying to load the users' profiles after logging out, at the time they are no longer authenticated and their profile is not loaded. Hence the error "**The user profile was not loaded as the user is not logged in.**". After logging out we were redirecting the users to "https://<env>.oidc.gov.bc.ca/auth/**info**" but it should've been "<env.loginRedirectLink/**info**>" instead. So for local, instead of "https://dev.oidc.gov.bc.ca/auth/info" we should redirect to "http://localhost:4200/info"
